examples/text-generation/run_generation.py

```diff
@@ -212,6 +212,11 @@ def setup_parser(parser):
         help="Store kv-cache in float8 when kv-cache is used",
     )
     parser.add_argument("--fp8", action="store_true", help="Enable Quantization to fp8")
+    parser.add_argument(
+        "--use_flash_attention",
+        action="store_true",
+        help="Whether to enable Habana Flash Attention.",
```

Comment:
Can we append the help text - "provided the model supports it" since we have added for llama only for now. See help text for attn_softmax_bf16.

Reply:
Added the text in second commit.